x-pack/plugins/security_solution/common/endpoint/schema/actions.ts

```diff
@@ -20,3 +20,10 @@ export const HostIsolationRequestSchema = {
     comment: schema.maybe(schema.string()),
   }),
 };
+
+export const EndpointActionLogRequestSchema = {
+  query: schema.object({}),
```

Comment:
can either leave this off this PR until we add the query definition, or at least add a comment saying this is for yet-to-come pagination query params